### PR TITLE
added non-spherical mask option to pytom_match_template.py

### LINKED ISSUE #21: non spherical mask option should be available in pytom_match_template.py
A non spherical mask flag should be added in the script: https://github.com/SBC-Utrecht/pytom-template-matching-gpu/blob/main/src/bin/pytom_match_template.py. It is already enabled in the code but forgot to yet add it as a flag to the script.

### PR DESCRIPTION
**Pull request by McHaillet**
Forgot to add this option to the script that was already implemented in the code.

Closes #21.

### CODE REVIEW

File: src/bin/pytom_match_template.py [hunk 1]
```diff
@@ -67,7 +70,7 @@ def main():
         args.mask,
         args.destination,
         angle_increment=args.angular_search,
-        mask_is_spherical=True,
+        mask_is_spherical=True if args.non_spherical_mask is None else args.non_spherical_mask,
```
Comment (sroet) [inline src/bin/pytom_match_template.py:73]: shouldn't this be
```suggestion
        mask_is_spherical=True if args.non_spherical_mask is None else (not args.non_spherical_mask),
```

**Review by sroet — CHANGES_REQUESTED** [review]
I think the logic is wrong, LGTM otherwise

**Comment by McHaillet** [comment]
> I think the logic is wrong, LGTM otherwise

indeed :+1:

**Review by sroet — APPROVED** [review]
LGTM, feel free to merge

**Pull request merged**